src/Provider/Provider.Web/Controllers/Part1/TrainingController.cs

```diff
@@ -48,12 +106,22 @@ public async Task<IActionResult> Training(TrainingEditModel m, [FromQuery] bool
             {
                 var vm = await _orchestrator.GetTrainingViewModelAsync(m, user);
                 vm.PageInfo.SetWizard(wizard);
+
```

Comment:
de-duplicate the VM code, as per previous suggestion

Reply:
:+1: